proto/rill/runtime/v1/api.proto

```diff
@@ -497,6 +512,30 @@ message GenerateMetricsViewFileResponse {
   bool ai_succeeded = 1;
 }
 
+message GenerateResolverRequest {
+  string instance_id = 1 [(validate.rules).string = {pattern: "^[_\\-a-zA-Z0-9]+$"}];
+  string prompt = 2;
+  string table = 3;
+  string connector = 4;
```

Comment:
nit: define `connector` before `table` – that's the order in all other protos with these two props